test/bats/test.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+load helpers
+
+BATS_TESTS_DIR=test/bats/tests
+WAIT_TIME=120
+SLEEP_TIME=1
+
+@test "osm pod has been deployed and is running." {
+    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get deployment/osm-controller"
```

Comment:
Instead of just checking on the deployment, we should check for the pod condition to catch issues like the pod status is `CrashLoopBackOff`
```sh
run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl wait --for=condition=Ready --timeout=60s pod -l app=osm-controller"
  assert_success
```

Reply:
that makes sense, updating now